### ISSUE TITLE
Alt tags in div... masonry ?  Accessibility Error and WCAG Error in menu

### ISSUE THREAD

**Issue by ip2C** [KriesiMedia/enfold-library#25]
Error: Attribute alt not allowed on element div at this point.

**`<div class='av-masonry-image-container' style="background-image: url(https://qualitaetsoffensive-tei…morientierung.jpg);"  title="22-Sozialraumorientierung"   alt="Menükachel: Sozialraumorientierung" ></div>`**

It seems that the masonry element pushes alt tags in divs.  This should be cleared? especcially for AAA and for WCAG or BITV

**Comment by InoPlugs** [comment]
@ip2C 

Thanks. You are right (see https://stackoverflow.com/questions/4216035/css-background-image-alt-attribute). Only title attribute needed.

I removed it in next release 6.0.1.


If you want to test it:

Replace ...\enfold\config-templatebuilder\avia-shortcode-helpers\class-avia-masonry.php

with 

https://github.com/KriesiMedia/enfold-library/blob/master/temp_fixes/enfold_6_0/config-templatebuilder/avia-shortcode-helpers/class-avia-masonry.php

**Comment by ip2C** [comment]
Thanks - it seems the  new issue System with Github is a quite well decision.  As i checked the code I see it is quite better. 

So I saw the next role error:  

Error: An element with role=menuitem must be contained in, or owned by, an element with role=menubar or role=menu.
`
</a></li><li role="menuitem" id="menu-item-84515" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-privacy-policy menu-item-mega-parent  menu-item-top-level menu-item-top-level-8"><a hre
`

Maybe you have time to check

**Comment by InoPlugs** [comment]
@ip2C 

Checking https://kriesi.at/themes/enfold-2017/  the 4 entries containing  .menu-item-mega-parent have parent role=menu.

Could you provide an example where we can see it  - and maybe a link to the backend to check the menu structure you have ?

**Comment by InoPlugs** [comment]
added role=”menu” aria-orientation=”vertical” to av-burger-overlay div for 6.0.2
